app/assets/stylesheets/dashboard.scss

```diff
@@ -297,3 +302,21 @@ p.debate-status {
     }
   }
 }
+
+@-webkit-keyframes slideIn {
```

Comment:
wouldn't this work without prefixing? (or should we also add prefix for firefox?)